docs/deployment.rst

```diff
@@ -169,58 +169,27 @@ Kafka & ZooKeeper
-
-Setup
-^^^^^
-
-Assuming you have Schema Registry running, here are the recommended steps to add Schema Registry instances in a new "slave" datacenter (call it DC B):
-
-- In DC B, make sure Kafka has ``unclean.leader.election.enable`` set to false.
-
-- In Kafka in DC B, create the ``_schemas`` topic. It should have 1 partition, ``kafkastore.topic.replication.factor`` of 3, and ``min.insync.replicas`` at least 2.
-
-- In DC B, run Replicator with Kafka in the "master" datacenter (DC A) as the source and Kafka in DC B as the target.
-
-- In the Schema Registry config files in DC B, set ``kafkastore.connection.url`` and ``schema.registry.zk.namespace`` to match the instances already running, and set ``master.eligibility`` to false.
-
-- Start your new Schema Registry instances with these configs.
-
-Run Book
-^^^^^^^^
-
-Let's say you have Schema Registry running in multiple datacenters, and you have lost your "master" datacenter; what do you do? First, note that the remaining Schema Registry instances will continue to be able to serve any request which does not result in a write to Kafka. This includes GET requests on existing ids and POST requests on schemas already in the registry.
-
-- If possible, revive the "master" datacenter by starting Kafka and Schema Registry as before.
-
-- If you must designate a new datacenter (call it DC B) as "master", update the Schema Registry config files so that ``kafkastore.connection.url`` points to the local ZooKeeper, and change ``master.eligibility`` to true. The restart your Schema Registry instances with these new configs in a rolling fashion.
+Migration from Zookeeper master election to Kafka master election
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is not required to migrate from Zookeeper based election to Kafka based master election. If
+you choose to do so, you need to make the below outlined config changes as the first step in all
+Schema Registry nodes
+
+- Remove ``kafkastore.connection.url``
+- Remove ``schema.registry.zk.namespace`` if its configured
+- Configure ``kafkastore.bootstrap.servers``
+
+After that, you need to shut down all the nodes and then start all the nodes starting
+from master eligible nodes. Please note that this would lead to temporary down time for
+Schema Registry.
+
+If you would like to avoid complete down time, you can do so by having downtime
+for write operations only. The steps for that are outlined below
+
+- Shutdown all Schema registry nodes except the master
+- Make above outlined config changes on that node and also ensure ``master.eligibility`` is set to false in each one of them
+- Start all the nodes on which the changes are made. These nodes will not be aware of the master since they are not aware of the ZK based master. All writes that come into these nodes will fail but reads will be successful.
```

Comment:
Good point, it might be easier to just keep one node up with master.eligibility=false( avoid multiple master) and then bring up the rest of nodes with new configs. After that apply, the configs to the one node and bounce it.